src/pages/SpacesDashboard/SpacesDashboard.tsx

```diff
@@ -32,9 +32,12 @@ export const SpacesDashboard: React.FC<SpacesDashboardProps> = ({
   const isWorldAdmin = userId ? world?.owners.includes(userId) : undefined;
   const { spaces } = useWorldSpaces({ worldId: world.id });
 
-  const renderedMapCards: JSX.Element[] | undefined = useMemo(
-    () =>
-      spaces?.filter(isPartyMapVenue).map((space) => {
+  const managedSpaces = spaces.filter(({ managedBy }) => !!managedBy);
+  const unmanagedSpaces = spaces.filter(({ managedBy }) => !managedBy);
```

Comment:
Hm, maybe we can replace this with a reduce that has 2 props in an object? Something like:
`{managedSpaces: [], unmanagedSpaces: []}` and then push into corresponding one based on the `managedBy` value?
Again, we'd avoid going through the space array twice this way